src/other/Table.js

```diff
@@ -759,8 +805,8 @@
         }
      };
 
-    Table.prototype.click = function (row, column) {
-        console.log("Click:  " + JSON.stringify(row) + ", " + column);
+    Table.prototype.click = function (row, column, selected) {
+        console.log("Click:  " + JSON.stringify(row) + ", " + column + "," + selected);
```

Comment:
@GordonSmith I replied to your comment in rowToObj() Widget.js, and removed my changes, so the comment is hidden.  The reason I made that change was for just for testing.  In the click event JSON.stringify returns a circular structure error when a widget is included. Obviously rowToObj() needs to return the widget itself, so the question is really how should I handle the logging of the click event?